app/lib/page_client.rb

```diff
@@ -1,27 +1,13 @@
 # frozen_string_literal: true
 
 require "xml_util"
-require "one_sky"
 
 class PageClient
   def initialize(resource, language_code)
     @resource = resource
     @language_code = language_code
   end
 
-  # I don't think it's critical to push all these files every time a new language is added, but we might as well do so
-  # because that will help keep OneSky up to date with the pages and phrases stored in the database.
-  def push_new_onesky_translation(keep_existing_phrases = true)
-    push_resource_pages(keep_existing_phrases)
-    push_name_description
-    push_translatable_attributes if @resource.uses_onesky?
-
-    self.class.delete_temp_pages
-  rescue => e
-    self.class.delete_temp_pages
-    raise e
-  end
-
   def self.delete_temp_pages
```

Comment:
Looks like this can go away, too - which would leave only delete_temp_dir, a one-liner that could be inlined the one place it's called.